generator/source/faq/grid.rst

```diff
@@ -2,46 +2,129 @@
 layout: page
 hide: true
 tags: [faq]
-title: Grid FAQ - Hyperledger Grid in General
+title: General FAQ
 permalink: /faq/grid/
 # Copyright (c) 2018-2019, Bitwise IO, Inc.
 # Copyright (c) 2018, Intel Corporation.
 # Licensed under Creative Commons Attribution 4.0 International License
 # https://creativecommons.org/licenses/by/4.0/
 ---
-Grid FAQ: Hyperledger Grid in General
-=====================================
-.. class:: mininav
 
-PREVIOUS_ FAQ_
+General Hyperledger Grid FAQ
+============================
 
 .. contents::
 
+.. _faq-what-is-hyperledger-grid:
 
-What is Grid?
--------------
+What is Hyperledger Grid?
+-------------------------
 
-Hyperledger Grid is a project for building supply chain solutions. It includes
-a set of libraries, data models, and SDKs to accelerate development for supply
-chain smart contracts and client interfaces.
+Hyperledger Grid is a platform for building supply chain solutions that include
+distributed ledger components. This project's goal is to speed up the development
+of modular blockchain-based solutions for cross-industry supply chain problems.
 
-What are some useful Grid links?
---------------------------------
+Hyperledger Grid provides a set of libraries, data models, and SDKs that can be
+used to develop supply chain smart contracts and client interfaces.
 
-GitHub repository for Grid
-    https://github.com/hyperledger/grid
-Grid documentation:
-    https://grid.hyperledger.org/docs/
-Grid chat main channel
-    https://chat.hyperledger.org/channel/grid
-Grid mailing list (not as popular as the chat channel above)
-    https://lists.hyperledger.org/g/grid
-Grid FAQ
-    https://grid.hyperledger.org/faq/
+For more information:
 
-.. class:: mininav
+* `About Hyperledger Grid </about>`__
 
-PREVIOUS_ FAQ_
+* `Hyperledger Grid project overview <https://www.hyperledger.org/projects/grid>`__
+  at hyperledger.org
+
+* `Hyperledger Grid wiki <https://wiki.hyperledger.org/display/grid>`__
```

Comment:
Do we want to point at the wiki? It not something we advertise elsewhere and is very bare bones.

Reply:
I'd be fine with it either way, but my observation is that this is a list entitled "For more information:" and the Hyperledger project page and wiki page provide no additional or interesting information.

Reply:
I dropped both the wiki and the project overview.  In their place, I added a link to the docs (which will have an intro and Grid overview soon).